tensorboard/webapp/widgets/data_table/data_table_test.ts

```diff
@@ -862,25 +862,5 @@ describe('data table', () => {
         contextMenu.nativeElement.innerHTML.includes('No Actions Available')
       ).toBeTrue();
     });
-
-    it('closes when the + button is clicked', () => {
```

Comment:
This was failing but actual functionality being tested is covered by the test which ensures there is never more than a single modal so this should be safe to delete.